server/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -200,16 +198,11 @@ private Result redirectToDashboardWithError(
                 Optional.empty(),
                 /* paramName=  page */
                 Optional.empty()))
-        .flashing("error", errorMessage)
-        .flashing("providedFirstName", form.get().getFirstName())
```

Comment:
Blocking (I think): Won't remove these from the flash prevent re-rendering the form with the old values populated? (Used in TrustedIntermediaryDashboardView)

Reply:
@swatkat1 was this addressed?